wiki/Gameplay/Game_modifier/Perfect/en.md

```diff
@@ -37,10 +37,10 @@ Any of the following acts **will cause** the Perfect mod to restart the beatmap:
 
 Any of the following acts **will not cause** the Perfect mod to restart the beatmap:
 
-- Obtaining a `300`
+- Obtaining a GREAT
```

Comment:
unless I'm missing something, this is still called a 300 in all the other game modes

Reply:
[yeah](https://github.com/ppy/osu-wiki/pull/8245#discussion_r1002804800)